sdk/identity/Azure.Identity/tests/Azure.Identity.Tests.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
     <PackageReference Include="BenchmarkDotNet" />
+    <PackageReference Include="Azure.Security.KeyVault.Secrets" />
```

Comment:
Why is this required?

Reply:
Found it

Reply:
it's required for the tests because we use Key Vault to validate that the tokens we get are valid.